cfgmgr/buffermgr.cpp

```diff
@@ -135,9 +135,10 @@ Create/update two tables: profile (in m_cfgBufferProfileTable) and port buffer (
 */
 task_process_status BufferMgr::doSpeedUpdateTask(string port, bool admin_up)
 {
-    vector<FieldValueTuple> fvVectorPg, fvVectorProfile;
     string cable;
     string speed;
+    string pfc_enable;
+    string applied_lossless_pg_status;
```

Comment:
`applied_lossless_pg_status` can be removed since it is no longer used.

Reply:
Good catch, will update.